src/plans/primal_dual_plan.jl

```diff
@@ -1,3 +1,5 @@
+abstract type AbstractPrimalDualProblem{T} <: Problem{T} end
```

Comment:
Please add a doc string here as well and see where you best can add that in the documentation (`Problem.md`) as well.